SharpDenizenTools/MetaObjects/MetaTypeData.cs

```diff
@@ -0,0 +1,51 @@
+﻿using FreneticUtilities.FreneticExtensions;
+using SharpDenizenTools.MetaHandlers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SharpDenizenTools.MetaObjects
+{
+    /// <summary>A generic-less <see cref="MetaTypeData{T}"/> interface.</summary>
+    public interface IMetaTypeData
+    {
+        /// <summary>Getter to create a new instance of this meta type.</summary>
+        Func<MetaObject> Getter { get; }
```

Comment:
This is a getter-getter, it should probably just be a singular getter

Reply:
Removed in favor of implementing `CreateNewMeta` in `MetaTypeData` (instead of the interface)